spec/models/contact_spec.rb

```diff
@@ -0,0 +1,35 @@
+# == Schema Information
+#
+# Table name: contact
+#
+#  id          :bigint(8)        not null, primary key
+#  provider_id :integer          not null
+#  type        :text
+#  name        :text
+#  email       :text
+#  telephone   :text
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+
+require 'rails_helper'
+
+describe Contact, type: :model do
+  it { should belong_to(:provider) }
+
+  describe 'type' do
+    it 'is an enum' do
+      expect(subject)
+        .to define_enum_for(:type)
```

Comment:
Is there value in testing this? Feels like testing config, but I'm not sure.

Reply:
This is testing config that is checked-in in the code. I'd be against testing config that's changeable outside the system.